src/main/java/gov/nasa/jpl/mbee/mdk/mms/sync/mms/MMSDeltaProjectEventListenerAdapter.java

```diff
@@ -248,9 +248,9 @@ private void determineChangesUsingCommitResponse(File responseFile, Set<String>
                 int size = 0;
                 String commitSyncDirection = "";
                 for(ObjectNode jsonObject : commitObjects) {
-                    if(jsonObject.isArray() && jsonObject.size() > 0) {
-                        for(int i = 0; i < jsonObject.size(); i++) {
-                            JsonNode currentNode = jsonObject.get(i);
+//                    if(jsonObject.isArray() && jsonObject.size() > 0) {
```

Comment:
Is this change in response to an API change in MMS? If so, please provide an example of the what the response looked like before and what it looks like now.

Also let's remove instead of commenting out and fix the indentation of the nested section.

Reply:
Note this is all new MMS4 code. It is unclear if it ever worked